src/Filters/molecules/DatePicker.tsx

```diff
@@ -359,22 +359,30 @@ const DatePicker: React.FC<IDatePicker> = ({
 
   const updateStartDate = useCallback((start: Date) => {
     const { end } = selectedRange ? selectedRange : TODAY_INTERVAL;
-    if (isAfter(add(start, { minutes: 1 }), end)) return;
+    if((isAfter(add(start, { minutes: 1 }), end))){
+      setCheckTimeValidation(true);
+    } else {
+      setCheckTimeValidation(false);
+    }
     setSelectedRange({ start, end });
   }, [selectedRange]);
 
   const updateEndDate = useCallback((end: Date) => {
     const { start } = selectedRange ? selectedRange : TODAY_INTERVAL;
-    if (isAfter(add(start, { minutes: 1 }), end)) return;
+    if(isAfter(add(start, { minutes: 1 }), end)){
+      setCheckTimeValidation(true);
```

Comment:
If need a return then please add that here as well.

Reply:
Here we are showing red border for input box when this condition will TRUE. so no need to return.

Reply:
ok